src/containers/CreatePipelineResource/CreatePipelineResource.js

```diff
@@ -64,38 +62,32 @@ export /* istanbul ignore next */ class PipelineResourcesModal extends Component
     };
   }
 
-  componentDidUpdate(prevProps) {
```

Comment:
Are we missing some realtime updates now ?

Reply:
No we're not missing anything, we should never have been requesting ServiceAccounts here as they're not used at all. My best guess is that it was copied along with the structure from another container and never removed.